components/Header/index.js

```diff
@@ -15,13 +22,15 @@ const Header = ({ backgroundColor, imgSrc, imgAlt, title, caption }) => {
         " md:h-1/2 max-w-md mx-auto overflow-hidden md:max-w-full"
       }
     >
-      <div className="md:flex md:justify-between">
-        <div className="p-8 lg:p-20 flex flex-col justify-between">
+      <div className="md:flex md:justify-around">
+        <div className="p-8 lg:p-20 flex flex-col justify-around">
           <h2 className="font-extrabold tracking-wide text-3xl md:text-6xl">
             {title}
           </h2>
           <p className="mt-2 text-gray-500 text-xl md:text-2xl">{caption}</p>
-          <Button caption="View Categories" onClick={handleClick} />
+          {buttonCaption ? (
+            <Button caption={buttonCaption} onClick={handleClick} />
+          ) : null}
```

Comment:
Whenever you have a ternary like this Negin, it's possible to replace the `?` with `&&` and then remove the `: null` part. It will work in exactly the same way if it looks like:
`{buttonCaption && <Button caption={buttonCaption} onClick={handleClick} />}`